### PR TITLE
🌐 Update Chinese translation for `docs/zh/docs/reference/*`

### PR DESCRIPTION
**Pull request by binaryYuki**

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit a4f72942bddda8e7ec33711bf2bd4a5ad7afbd6d at: https://ea5d6a24.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 25e586c3bf0ffb7fab44f8aa5a5b727cf00a00d5 at: https://1eab456e.fastapitiangolo.pages.dev

File: docs/zh/docs/reference/index.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# API参考
+
+这里有代码 API、类、函数、参数、属性和您可以在应用程序中使用的所有 FastAPI 部件的参考说明。
+
+如果您想**学习FastAPI**，最好阅读
+[FastAPI Tutorial](https://fastapi.tiangolo.com/tutorial/).
```
Comment (nick-cjyx9) [inline docs/zh/docs/reference/index.md:6]: fix: https://fastapi.tiangolo.com/zh/

File: docs/zh/docs/reference/middleware.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# 中间件
+
+Starlette 可直接提供多个中间件。
+
+请参阅 [FastAPI docs for Middleware](https://fastapi.tiangolo.com/advanced/middleware/).
```
Comment (nick-cjyx9) [inline docs/zh/docs/reference/middleware.md:5]: https://fastapi.tiangolo.com/zh/advanced/middleware/

File: docs/zh/docs/reference/responses.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
+# 自定义响应 - 文件、HTML、重定向、流等
+
+您可以使用多个自定义响应类来创建实例，并直接从 *path operations* 中返回。
+
+请参阅 [FastAPI docs for Custom Response - HTML, Stream, File, others](https://fastapi.tiangolo.com/advanced/custom-response/).
```
Comment (nick-cjyx9) [inline docs/zh/docs/reference/responses.md:5]: https://fastapi.tiangolo.com/zh/advanced/custom-response/

File: docs/zh/docs/reference/staticfiles.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# 静态文件 - `StaticFiles`
+
+您可以使用 `StaticFiles` 类提供 JavaScript、CSS、图片等静态文件。
+
+请参阅 [FastAPI docs for Static Files](https://fastapi.tiangolo.com/tutorial/static-files/).
```
Comment (nick-cjyx9) [inline docs/zh/docs/reference/staticfiles.md:5]: 以下链接问题均同上

**Review by nick-cjyx9 — CHANGES_REQUESTED** [review]

File: docs/zh/docs/reference/staticfiles.md (hunk 4 above)
Comment (binaryYuki) [reply docs/zh/docs/reference/staticfiles.md:5]: 已修改

Comment (binaryYuki) [reply docs/zh/docs/reference/staticfiles.md:5]: > 以下链接问题均同上

已修改

**Comment by tiangolo** [comment]
📝 Docs preview for commit 43d3386b73b0f04f5da04567a4bfff22055002d1 at: https://b1b2ab39.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit e1d39be0e522a8ed5174afc45fc3fe347ab93ab8 at: https://a73f9e1a.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 3a9f7ae929f79ff2ee835bfff86b10aa795bfee9 at: https://0d2fec8d.fastapitiangolo.pages.dev

**Pull request closed without merging**